src/main/resources/liquibase/crop_changelog/21_4_0.xml

```diff
@@ -0,0 +1,144 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+				   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+				   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+		http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+	<changeSet author="cuenyad" id="v21.4.0-1">
+			<preConditions onFail="MARK_RAN">
+				<and>
+					<sqlCheck expectedResult="0">select count(*) from cvterm where name = 'Female Parent' and cv_id = 1010 and is_obsolete = 0 and cvterm_id = 8341;</sqlCheck>
+				</and>
+			</preConditions>
+			<comment>Add 'Female Parent' property</comment>
+			<sql dbms="mysql" splitStatements="true">
+				INSERT INTO CVTERM (cvterm_id, cv_id, name, definition, is_obsolete, is_relationshiptype, is_system)
+				VALUES (8341, 1010, 'Female Parent', 'Female Parent', 0, 0, 0);
+
+				INSERT INTO cvterm_relationship(type_id, subject_id, object_id)
+				VALUES (1225, 8341, 1087);
+			</sql>
+	</changeSet>
+
+	<changeSet author="cuenyad" id="v21.4.0-2">
+			<preConditions onFail="MARK_RAN">
+				<and>
+					<sqlCheck expectedResult="0">select count(*) from cvterm where name = 'Male Parent' and cv_id = 1010 and is_obsolete = 0 and cvterm_id = 8344;</sqlCheck>
+				</and>
+			</preConditions>
+			<comment>Add 'Male Parent' property</comment>
+			<sql dbms="mysql" splitStatements="true">
+				INSERT INTO CVTERM (cvterm_id, cv_id, name, definition, is_obsolete, is_relationshiptype, is_system)
+				VALUES (8344, 1010, 'Male Parent', 'Male Parent', 0, 0, 0);
+
+				INSERT INTO cvterm_relationship(type_id, subject_id, object_id)
+				VALUES (1225, 8344, 1087);
+			</sql>
+	</changeSet>
+
+	<changeSet author="cuenyad" id="v21.4.0-3">
+			<preConditions onFail="MARK_RAN">
+				<and>
+					<sqlCheck expectedResult="0">select count(*) from cvterm where name = 'FEMALE_PARENT_NAME' and cv_id = 1040 and is_obsolete = 0;</sqlCheck>
+					<sqlCheck expectedResult="0">select count(*) from cvterm where cvterm_id = 8236;</sqlCheck>
+					<sqlCheck expectedResult="1">select count(*) from cvterm where name = 'Female Parent' and cv_id = 1010 and is_obsolete = 0 and cvterm_id = 8341;</sqlCheck>
+					<sqlCheck expectedResult="1">select count(*) from cvterm_relationship where type_id = 1225 and subject_id = 8341 and object_id = 1087;</sqlCheck>
```

Comment:
@cuenyad not following why this last sqlCheck is really needed?

Reply:
done!